src/containers/App/index.js

```diff
@@ -102,6 +103,7 @@ export const AppContainer = () => {
               <Route path="/instellingen" component={SettingsModule} />
               <Route path="/incident" component={IncidentContainer} />
               <Route path="/kto/:satisfactionIndication/:uuid" component={KtoContainer} />
+              <Route path="/category/:category/:subcategory" component={IncidentClassification} />
```

Comment:
Wel netjes om het URL Nederlands te houden

Reply:
incident is not Nederlands :-). Should I change that too?

Reply:
done

Reply:
done